### ISSUE TITLE
Require options

### ISSUE THREAD

**Issue by Ismael-Oliv** [atom-ide-community/atom-ide-javascript#4]
When I try to import using javascript like this way "const fs = require('fs') and typescript with "import", it doesn't show all the "require" options that it used to do.
I'm using the 1.2.0 atom-ide-javascript version

[video.zip](https://github.com/atom-ide-community/atom-ide-javascript/files/4976043/video.zip)

**Comment by aminya** [comment]
Does it show any other autocomplete? Something like:
```
["1"].conc
```
should suggest `concat`.

**Comment by Ismael-Oliv** [comment]
yeah, It does 

![Captura de tela de 2020-07-25 14-18-37](https://user-images.githubusercontent.com/63047946/88462490-0a2ccd00-ce82-11ea-8363-7de79a0aacba.png)

**Comment by aminya** [comment]
@lierdakil Any idea about this? Has anything changed from the TypeScript side?

**Comment by aminya** [comment]
@Ismael-Oliv Please try with v1.2.1 and see if this fixes it.

**Comment by Ismael-Oliv** [comment]
Not yet, but it worked when I installed the express types' => @types/express.

![Captura de tela de 2020-07-25 14-58-47](https://user-images.githubusercontent.com/63047946/88463277-fbe1af80-ce87-11ea-8e4c-539f9bb446b7.png)

And I think it's because of autocomplete-paths package , its showing so many stuffs .

![Captura de tela de 2020-07-25 15-04-01](https://user-images.githubusercontent.com/63047946/88463298-17e55100-ce88-11ea-8e4a-ad903722ce41.png)

**Comment by aminya** [comment]
If you type 
```
const fs = require('f ')
```
doesn't it offer 'fs'?

**Comment by Ismael-Oliv** [comment]
yeah, But it just shows the "require" options because I had installed the express types of typescript

![Captura de tela de 2020-07-25 15-11-13](https://user-images.githubusercontent.com/63047946/88463614-5714a180-ce8a-11ea-9159-befcd1987881.png)

When I try to import anything without installing any library, it's not showing ...

![Captura de tela de 2020-07-25 15-18-48](https://user-images.githubusercontent.com/63047946/88463597-449a6800-ce8a-11ea-83ff-188c2091d11c.png)
 
But I'm gonna remove atom and Install again with all packages. to see if it work

**Comment by aminya** [comment]
OK. Now I understand the problem better. When you do not have any library installed, it does not show it. Although I cannot reproduce it, I will take a closer look.
![image](https://user-images.githubusercontent.com/16418197/88463786-09446d00-ce7b-11ea-832b-fb497822891c.png)

**Comment by aminya** [comment]
I can confirm that sometimes the autocomplete is not fast enough here. It might be because of the TypeScript server. @lierdakil and I will take a look into this.

**Comment by Ismael-Oliv** [comment]
I installed it again and it's not working yet

![Captura de tela de 2020-07-25 15-56-38](https://user-images.githubusercontent.com/63047946/88464252-9b567080-ce8f-11ea-9696-3a85355405ca.png)

if is there something that I can do to help you, just tell me.

**Comment by aminya** [comment]
Try installing `@types/node` for now until we find a way.

**Comment by Ismael-Oliv** [comment]
Alright, no problem. :+1:

**Comment by aminya** [comment]
I think I should add an option for automatic Nodejs completions. Maybe it works on my system out of the box because I have it installed globally.

**Comment by Ismael-Oliv** [comment]
Alright, I'm gonna try it ...

**Comment by lierdakil** [comment]
Without `@types/*` packages in scope, TypeScript server should generally recognize `require`, but only offers sub-directories of `node_modules` as "bare" import suggestions. If you add `./` to the argument, it switches to local files instead. Generally a good idea to add relevant `@types` to `devDependencies` in any case, although I appreciate it can be a bit of a hassle. Also, unless `require` is defined somewhere in the ambient declaration,  tsserver doesn't assume `require` is even defined, so it doesn't offer _it_ as a suggestion.

There is a caveat however, TypeScript will sometimes cache type definitions in `$HOME/.cache/typescript` and might use those even if there aren't any installed in the current project. There is very little information on when exactly this happens however, so I couldn't tell you how that works if my life depended on it.

Including/enabling Node definitions by default isn't the greatest idea, because some of us write code for the browser (shocking, I know), and we don't really want to see Node-specific suggestions that won't work.

**Comment by aminya** [comment]
I have a suggestion. We can add a command and a Tool-bar button to add Nodejs support by using that. People who need Node should click or run that command first, or should install the @types

**Comment by lierdakil** [comment]
Typing `npm i --save-dev --only=dev @types/node` once per project isn't that much of a chore I reckon, so I don't see much value in inventing new ways to add some definitions (which aren't even necessarily correct, because who knows what node version does the user intend to use) only for the sake of saving a couple keystrokes.

The "hassle" I was talking about above is tracking down and installing definitions for _all_ the dependencies -- that _is_ a bit of a chore.

**Comment by aminya** [comment]
I am planning to add a npm IDE which adds GUI and clickable buttons for running various operations. I think we can address installing the types in that package!
